code/game/objects/structures/displaycase.dm

```diff
@@ -13,6 +13,7 @@
 	var/obj/item/showpiece = null
 	var/alert = 0
 	var/open = 0
+	var/openable = 1

```

Comment:
Use TRUE/FALSE defines please. Also if you could make the old variables use them please.